packages/server/src/services/auth/createUser.ts

```diff
@@ -38,7 +41,7 @@ const createUser = async (req, res, _next, userData) => {
   const username = sanitiseUsername(userData.email.split("@")[0].slice(0, 30));
 
   // get avatar by hashing email
-  const userMd5Hash = md5(userData.email);
+  const userMd5Hash = md5(lowerCaseEmail);
```

Comment:
```suggestion
  const userMd5Hash = md5(email);
```

Reference to https://github.com/logchimp/logchimp/pull/1073/files#r2284514871